Cargo.toml

```
@@ -4,7 +4,7 @@ members = [
     "agent/resource-agent",
     "agent/test-agent",
     "bottlerocket/agents",
-    "bottlerocket/testsys",
+    "testsys",
```

I'm open to suggestions as far as naming goes, but I think we should not use `testsys` as the name of this CLI. I don't want there to be confusion between the CLI we create in the Bottlerocket repo and the CLI in the `bottlerocket-test-system` repo. It needs to be clear that they are not the same thing!

```suggestion
    "cli",
```